src/gui/MainWindow.cpp

```diff
@@ -765,24 +765,24 @@ void MainWindow::openFile(const QString& filePath) {
 }
 
 bool MainWindow::saveFile(const QString& filePath) {
-    if (activeDocumentId == -1) return false;
     return documents[activeDocumentId]->getDatabase()->Save(filePath.toUtf8().data());
 }
 
-void MainWindow::openFileDialog()
-{
+void MainWindow::openFileDialog() {
 	const QString filePath = QFileDialog::getOpenFileName(documentArea, tr("Open BRL-CAD database"), QString(), "BRL-CAD Database (*.g)");
-    if (!filePath.isEmpty()){
+    if (!filePath.isEmpty()) {
         openFile(filePath);
     }
 }
 
 void MainWindow::saveAsFileDialog() {
-    if (activeDocumentId == -1) return;
+    if (activeDocumentId == -1) {
+        return;
+    }
+
```

Comment:
This doesn't change anything but clutters the patch.

Reply:
I forgot to remove those styles. I remove it in next commit.